.github/workflows/release-notes.yml

```diff
@@ -0,0 +1,25 @@
+name: Release Notes
+
+on:
+  push:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        compiler: [clang]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Environment setup
+      uses: actions/checkout@v2
+    - name: Check release notes
+      env:
+        CC: ${{ matrix.compiler }}
+        FC: gfortran-9
+      run: |
+        git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin/main
+        git checkout main && git pull && git checkout -
```

Comment:
```suggestion
        git fetch origin main
```

Reply:
Oh, or use this syntax in the fancier invocation above.

Reply:
Thanks, this isn't a way I've used Git very much